packages/sarif-to-markdown/test/snapshots/hidden-details/output.md

```diff
@@ -22,7 +22,7 @@
     - js/xss [error] 
 
     > Client-side cross-site scripting
-
+</details>
```

Comment:
After fixing the tests so all asserted on the markdown output, I deleted the failing tests output.md and then ran tests with UPDATE_SNAPSHOT. The only difference in the new snapshots were the ending </details> which makes logical syntactic sense.